tools/migrations/add_img_url_to_article.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE
+    `zeeguu_test`.`article`
+ADD
+    COLUMN `img_url_id` INT NULL
+AFTER
+    `deleted`,
+ADD
+    UNIQUE INDEX `img_url_id_UNIQUE` (`img_url_id` ASC) VISIBLE;
```

Comment:
VISIBLE seems to not be supported by the MySQL engine on the server! if you simply drop it, the query executes.